commands/cli.go

```diff
@@ -22,6 +23,10 @@ import (
 func CreateCli(version string, mixPanelClientId string) *cli.App {
 	app := cli.NewApp()
 	logging.InitLogger("cloud-nuke", version)
+	_, disableTelemetryFlag := os.LookupEnv("DISABLE_TELEMETRY")
+	if !disableTelemetryFlag {
+		fmt.Println("This program sends telemetry to Gruntwork. To disable, set DISABLE_TELEMETRY=true as an environment variable")
```

Comment:
Should this be STDERR instead of STDOUT?

Reply:
hmm ya that's probably a good idea, I'll send it through the standard logger as well so it gets json formatted if that's configured